docs/core/additional-tools/dotnet-svcutil-guide.md

```diff
@@ -45,73 +52,115 @@ mkdir HelloSvcutil
 cd HelloSvcutil
 ```
 
```

Comment:
on line 44 can you make the following change?
the web service will be assumed to be hosted at the following address ->
let's assume the web service will be hosted at the following address